setup-common-project-files.sh

```diff
@@ -28,13 +32,15 @@ isDebugConfiguration=false
 # Methods
 #
 
-function display_usage () { 
-	echo "This script performs all common project setup scripts by default. You can optionally pass the projects base directory path as argument. Exceptions can be declared with the flags:" 
-	echo -e "$noSwiftlintFlag" 
-	echo -e "$noCodebeatFlag" 
-	echo -e "\nUsage:\n$ $0 $noCodebeatFlag\n" 
-	echo -e "or:\n$ $0 $noCodebeatFlag /Code/Project/Test\n" 
-} 
+function display_usage () {
+	echo "This script performs all common project setup scripts by default. You can optionally pass the projects base directory path as argument. Exceptions can be declared with the flags:"
+	echo -e "$noSwiftlintFlag"
+	echo -e "$frameworkFlag"
```

Comment:
`frameworkFlag` is no exception. I think you should modify the usage string to keep the opt-out flags and add other parameters with a new text.